PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -77,9 +78,11 @@ public void onClick(DialogInterface dialog, int id) {
                     drawable.setAlpha(200);
                     dialog.getWindow().setBackgroundDrawable(drawable);
                     dialog.show();
+                    SessionHistory.hasPreviouslyCustomized = false;
                 } else{
                     Intent intent = new Intent(getApplicationContext(),AvatarRoomActivity.class);
                     startActivity(intent);
+                    SessionHistory.hasPreviouslyCustomized = false;
```

Comment:
The statement SessionHistory.hasPreviouslyCustomized = false has been included in both the if block and else block. Why not include this once at the end outside the if-else block?

Reply:
@Rimjhim28 any response to this inquiry?